docs/guide/examples.rst

```diff
@@ -152,61 +152,94 @@ If your callback returns False, training is aborted early.
 
 .. code-block:: python
 
-  import os
-
-  import gym
-  import numpy as np
-  import matplotlib.pyplot as plt
-
-  from stable_baselines.ddpg.policies import LnMlpPolicy
-  from stable_baselines.bench import Monitor
-  from stable_baselines.results_plotter import load_results, ts2xy
-  from stable_baselines import DDPG
-  from stable_baselines.ddpg import AdaptiveParamNoiseSpec
-
-
-  best_mean_reward, n_steps = -np.inf, 0
-
-  def callback(_locals, _globals):
-    """
-    Callback called at each step (for DQN an others) or after n steps (see ACER or PPO2)
-    :param _locals: (dict)
-    :param _globals: (dict)
-    """
-    global n_steps, best_mean_reward
-    # Print stats every 1000 calls
-    if (n_steps + 1) % 1000 == 0:
-        # Evaluate policy training performance
-        x, y = ts2xy(load_results(log_dir), 'timesteps')
-        if len(x) > 0:
-            mean_reward = np.mean(y[-100:])
-            print(x[-1], 'timesteps')
-            print("Best mean reward: {:.2f} - Last mean reward per episode: {:.2f}".format(best_mean_reward, mean_reward))
-
-            # New best model, you could save the agent here
-            if mean_reward > best_mean_reward:
-                best_mean_reward = mean_reward
-                # Example for saving best model
-                print("Saving new best model")
-                _locals['self'].save(log_dir + 'best_model.pkl')
-    n_steps += 1
-    return True
-
-
-  # Create log dir
-  log_dir = "/tmp/gym/"
-  os.makedirs(log_dir, exist_ok=True)
-
-  # Create and wrap the environment
-  env = gym.make('LunarLanderContinuous-v2')
-  env = Monitor(env, log_dir, allow_early_resets=True)
-
-  # Add some param noise for exploration
-  param_noise = AdaptiveParamNoiseSpec(initial_stddev=0.1, desired_action_stddev=0.1)
-  # Because we use parameter noise, we should use a MlpPolicy with layer normalization
-  model = DDPG(LnMlpPolicy, env, param_noise=param_noise, verbose=0)
-  # Train the agent
-  model.learn(total_timesteps=int(1e5), callback=callback)
+    import os
```

Comment:
For consistency, the code should be indented with two whitespaces like previously (and like in rest of the same document)